scipy/sparse/csgraph/tests/test_pydata_sparse.py

```diff
@@ -22,6 +23,15 @@
                  pytest.param("DOK", marks=[pytest.mark.xfail(reason=msg)]))
 
 
+def check_sparse_version(min_ver):
+    if sparse is None:
+        return pytest.mark.skip(reason="sparse is not installed")
```

Comment:
Nit, also wouldn't this be handled by `pytestmark` automatically? Or would we need this in addition?

```suggestion
        return pytestmark
```

Reply:
`pytestmark` applies its markers to all tests but we want to skip only this test based on the `sparse` version.

I would prefer to keep it as `return pytest.mark.skip(reason="sparse is not installed")`, as `pytestmark` is a different concept and doesn't fit here. It defines markers for every test function and it can be a list:
https://scipy.github.io/devdocs/dev/api-dev/array_api.html#adding-tests

Reply:
Yeah, since we already have the skip for `sparse is None` in `pytestmark`, we could probably do without adding this identical skip as well here. We do need the early return though, so that `sparse.__version__` exists below. I think it's fine to just add the extra skip mark.